app/src/main/java/at/roteskreuz/stopcorona/model/db/DefaultDatabase.kt

```diff
@@ -135,10 +135,15 @@ abstract class DefaultDatabase : RoomDatabase() {
             migration(12, 13) {
                 // add new tables
                 execSQL(
-                    "CREATE TABLE IF NOT EXISTS `received_infection_message` (`uuid` TEXT NOT NULL, `messageType` TEXT NOT NULL, `timeStamp` INTEGER NOT NULL, PRIMARY KEY(`uuid`))"
+                    """CREATE TABLE IF NOT EXISTS `received_infection_message` (
```

Comment:
If you decided to format older code, please format it in some reasonable way. Like one command per line, new line after comma or after opening bracket and before closing bracket new line etc.

Applied for SQL queries below as well.